bedrock-devnet/devnet/__init__.py

```diff
@@ -15,25 +15,77 @@
 
 pjoin = os.path.join
 
-parser = argparse.ArgumentParser(description='Bedrock devnet launcher')
-parser.add_argument('--monorepo-dir', help='Directory of the monorepo', default=os.getcwd())
-parser.add_argument('--allocs', help='Only create the allocs and exit', type=bool, action=argparse.BooleanOptionalAction)
-parser.add_argument('--test', help='Tests the deployment, must already be deployed', type=bool, action=argparse.BooleanOptionalAction)
-parser.add_argument('--l2', help='Which L2 to run', type=str, default='op1')
-parser.add_argument('--l2-provider-url', help='URL for the L2 RPC node', type=str, default='http://localhost:19545')
-parser.add_argument('--deploy-l2', help='Deploy the L2 onto a running L1 and sequencer network', type=bool, action=argparse.BooleanOptionalAction)
-parser.add_argument('--deploy-config', help='Deployment config, relative to packages/contracts-bedrock/deploy-config', default='devnetL1.json')
-parser.add_argument('--deploy-config-template', help='Deployment config template, relative to packages/contracts-bedrock/deploy-config', default='devnetL1-template.json')
-parser.add_argument('--deployment', help='Path to deployment output files, relative to packages/contracts-bedrock/deployments', default='devnetL1')
-parser.add_argument('--devnet-dir', help='Output path for devnet config, relative to --monorepo-dir', default='.devnet')
-parser.add_argument('--espresso', help='Run on Espresso Sequencer', type=bool, action=argparse.BooleanOptionalAction)
+parser = argparse.ArgumentParser(description="Bedrock devnet launcher")
+parser.add_argument(
+    "--monorepo-dir", help="Directory of the monorepo", default=os.getcwd()
+)
+parser.add_argument(
+    "--allocs",
+    help="Only create the allocs and exit",
```

Comment:
@jbearer our auto-formatters appear to be configured differently, if you let me know your suggests I'll adjust mine to yours

Reply:
I don't think I ran a formatter, so we can go with yours. How did you set it up?

Reply:
I use VSCode, it's just using the default [Black Formatter](https://github.com/psf/black)

Reply:
Actually, many of these changed lines are from the upstream optimism repo. So in the interest of keeping the changes in our fork to a minimum maybe it's better if we don't format this file?